science.tex

```diff
@@ -18,7 +18,7 @@ \subsection{Solar System}
 
 \subsection{Static Science}
  
-Datasets for static science will flow from the \svs carried out during commissioning and released as Data Preview 2 (DP2). 
+Datasets for static science will flow from the SV Surveys carried out during commissioning and released as Data Preview 2 (DP2). 
 The commissioning team are planning to acquire on-sky observations that would enable science validation studies for the four LSST science drivers.
 Guidance is being sought from the community to enhance opportunities for science validation and early science  based on commissioning data.
```

Comment:
This would be a natural place to reference the process of collecting "Commissioning Notes" from the community that took place in 2020. See https://community.lsst.org/t/community-input-to-the-on-sky-observing-strategy-during-commissioning/4406